src/lib/drivers/smbus/SMBus.cpp

```diff
@@ -141,11 +158,12 @@ int SMBus::block_write(const uint8_t cmd_code, void *data, uint8_t byte_count, b
 
 	// If block_write fails, try up to 10 times.
 	while (i < 10 && ((result = transfer((uint8_t *)buf, byte_count + 2, nullptr, 0)) != PX4_OK)) {
+		perf_count(_interface_errors);
 		i++;
 	}
 
 	if (i == 10 || result) {
-		PX4_WARN("Block_write failed %d times", i);
+		//PX4_WARN("Block_write failed %d times", i); removed, it potentially floating the shell
```

Comment:
Yes please remove things like this. If it's important enough we should be logging the errors in perf counters or the message itself.

Reply:
will do